lighthouse-service/event_handler/start_evaluation_handler.go

```diff
@@ -168,13 +168,29 @@ func (eh *StartEvaluationHandler) sendInternalGetSLIEvent(shkeptncontext string,
 		},
 	}
 
+	deployment := ""
+	if e.Deployment.DeploymentNames != nil && len(e.Deployment.DeploymentNames) > 0 {
```

Comment:
Please also add a `e.Deployment != nil`

Reply:
not necessary, as it is a "non-nillable" type.